transcom/mymove
# B 22148 main - Assigned User Cleanup [BE] - PPM Closeout

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-22148)

## Summary
Once an assigned office user has completed their task on  a move, we need to clean up that assigned and remove them from the move.

> [!IMPORTANT]
> This contains code from [B-21359](https://github.com/transcom/mymove/pull/14330)
> B-21359 should be merged first 

### How to test
1.  Create a PPM Move that needs Needs Closeout.
2. Assign an office user in the PPM Closeout queue
3. Check DB to verify the assigned users is assigned 
4. Review the documents for the PPM Move and approve
5. Check DB to verify the previously assigned users no longer assigned
